bundle/tests/scorecard/kuttl/runtime_pullpolicy/02-never.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: example-runtime-component
+spec:
+  applicationImage: navidsh/demo-day
+  replicas: 1
+  pullPolicy: "Never"
```

Comment:
Please add a test to verify that removing `spec.pullPolicy` from the `RuntimeComponent` CR, results in Deployment/Pod being created with the default pull policy values (in this case the image is referring to _latest_ tag implicitly, hence pull policy the k8s resource(s) should be _Always_ I believe).